sample-android/app/build.gradle.kts

```diff
@@ -1,15 +1,11 @@
-import com.louiscad.splitties.AndroidX
-import com.louiscad.splitties.Google
-import com.louiscad.splitties.Kotlin
-import com.louiscad.splitties.Testing
 import org.jetbrains.kotlin.gradle.tasks.KotlinCompile
 
 plugins {
     id("com.android.application")
-    id("com.louiscad.splitties")
     id("org.jetbrains.kotlin.android")
     id("org.jetbrains.kotlin.kapt")
-    id("kotlin-android-extensions")
+    id("org.jetbrains.kotlin.android.extensions")
+    id("gradle.site")
```

Comment:
Did you add the gradle.site plugin to provide an example of using a plugin with its id?

Reply:
Exactly